python/llm/setup.py

```diff
@@ -300,6 +300,12 @@ def setup_package():
     serving_requires = ['py-cpuinfo']
     serving_requires += SERVING_DEP
 
+    npu_requires = copy.deepcopy(all_requires)
+    cpu_transformers_version = ['transformers == 4.36.2', 'tokenizers == 0.15.2']
+    for exclude_require in cpu_transformers_version:
+        npu_requires.remove(exclude_require)
+    npu_requires += ["transformers==4.40.0",
+                     "bigdl-core-npu==" + CORE_XE_VERSION]
```

Comment:
Should we add `;platform_system=='Windows'"`  ?

Reply:
Do we need to raise an error when Linux users try to install `ipex-lllm[npu]`? @jason-dai